proto/info/ntpsources.proto

```diff
@@ -0,0 +1,74 @@
+// Copyright(c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package org.lfedge.eve.info;
+option go_package  = "github.com/lf-edge/eve-api/go/info";
+
+option java_package = "org.lfedge.eve.info";
+
+// Enum for describing the NTP source state
+enum NTPSourceState {
+  NTP_SOURCE_STATE_SYNC        = 0;
```

Comment:
This buflint should be fixed since zero needs to be reserved (by definiting an _UNSPECIFIED name) and the real values starting at 1.